routing_model/index.html

```diff
@@ -591,95 +611,109 @@ <h2>Deciding that a member of our section should be relocated away</h2>
 
-      RPC((RPC))
-      RPC --RefuseCandidate--> VoteParsecRefuseCandidate
-      RPC --RelocateResponse--> VoteParsecRelocateResponse
-      VoteParsecRefuseCandidate["vote_for(<br/>ParsecRefuseCandidate)<br/>to handle the RPC consistently"]
-      VoteParsecRefuseCandidate --> LoopEnd
-      VoteParsecRelocateResponse["vote_for(<br/>ParsecRelocateResponse)<br/>to handle the RPC consistently"]
-      VoteParsecRelocateResponse --> LoopEnd
-
-
       ParsecConsensus((Parsec<br/>consensus))
-      DiscardParsec["Discard<br/>Parsec<br/>event"]
-      Bug["BUG: assert fail:<br/><br/>Section can only send either<br/>one of these RPC, and only for<br/>the candidate we are relocating"]
-
       ParsecConsensus -- WorkUnitIncrement consensused --> IncrementWorkUnit
       IncrementWorkUnit["increment_nodes_work_units()"]
       IncrementWorkUnit-->AlreadyRelocating
       AlreadyRelocating(("Check?"))
 
 
-      AlreadyRelocating --"RELOCATING.is_none()"--> SetRelocating
-      AlreadyRelocating --"RELOCATING.is_some()"--> DiscardParsec
-      ParsecConsensus --"Any:<br/><br/>ParsecRefuseCandidate<br/><br/>ParsecRelocateResponse"--> Bug
-      Bug
-      DiscardParsec --> LoopEnd
-
-      SetRelocating["RELOCATING=Some(get_node_to_relocate())"]
-      SetRelocating --> SetRelocatingNodeState
-
-      SetRelocatingNodeState["set_node_state(RELOCATING, RelocatingState)"]
-      SetRelocatingNodeState --> isRelocatingElder
-
-      isRelocatingElder(("Check?"))
-      isRelocatingElder--"is_elder(RELOCATING)"-->ResetRelocating
-      isRelocatingElder--"!is_elder(RELOCATING)"-->Concurrent0
+      AlreadyRelocating --"nodes_relocating().is_empty()"--> SetRelocatingNodeState
+      AlreadyRelocating --"Otherwise"--> LoopEnd
 
-      Concurrent0{"Concurrent<br/>paths"}
-      Concurrent0 --> TryRelocating
-      Concurrent0 --> LoopEnd
-
-      TryRelocating["TryRelocating<br/>(shared state)"]
-      style TryRelocating fill:#f9f,stroke:#333,stroke-width:4px
-      TryRelocating-->ResetRelocating
-
-      ResetRelocating["RELOCATING=None<br/>(shared state)"]
-      ResetRelocating --> LoopEnd
+      SetRelocatingNodeState["set_node_state(get_node_to_relocate(), RelocatingState)"]
+      SetRelocatingNodeState --> LoopEnd
   </div>
 
   <h2>Relocating a member of our section away from it</h2>
   <div class=description>
-  <p>At this stage, we have decided that one of our section members should be relocated. The process is quite simple: we send an ExpectCandidate RPC to the destination section. That section will eventually either pass the node to a section with a shorter prefix, send us a RefuseCandidate RPC or eventually send us a RelocateResponse RPC. If the destination picked another section with a shorter prefix, that section will send us one of these RPCs (or suggest another section with a shorter prefix, but this recursion will end at some point).<br/>
+  <p>At this stage, we have decided that one of our section members should be relocated.<br/>
```

Comment:
I'll try to update.
What I was trying to say is this flow only cater to node we have decided to relocate, i.e it will do nothing unless a node is in RelocatingState.